externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/events/SparkOperationEvent.scala

```diff
@@ -67,7 +67,9 @@ case class SparkOperationEvent(
     sessionId: String,
     sessionUser: String,
     executionId: Option[Long],
```

Comment:
Also add an annotation for `executionId`.

Reply:
ok